notebooks/Emukit-multi-fidelity-bayesian-optimization.ipynb

```diff
@@ -0,0 +1,692 @@
7uvvtu+vTpw1dffUXDhg3p27dvltKhzHr37s3HH3/MU089xeTJk7nooovo1atXvp2jW7duzJo1C4DBgweTkJDApEmTfG57/Phxrr/+elauXMno0aP597//zcqVK3nttdeybPvwww/z8MMP06lTJ77++mteeeUV/vvf/3LDDTfg8XjO+JlTxcTEpN20xMfHk5CQQEJCAu3atWPQoEFs3bqVb7/9NsM+//nPfzh58iT33ntvjs4hIiIXjpQUWLYMtm8PrmQeYO3apSxcOIvBg1+mXLmK/g4n11Rykw9at26dZVmPHj0YPHgwx44do1OnTlnW9+7dm969e7N3716fyfGgQYO49dZb2bJlC3fddVeW9Y888gg33ngja9asoX79+vnyOc7k0KFDTJo0ibi4uLRlJ06cyNG+Ho+HIUOGEBcXxxdffJG2vFWrVtSqVYt33nknSyt/ZgcPHuR///sfzZo1A6B58+bMnDmTTz/9NG1ZZsuWLePLL7/knXfe4cEHHwSgXbt2JCYm8q9//StfzlGhQgUiIyMBV+5yppKlDz74gC1btrBo0SKuuOIKADp27Jjl+1u7di3/+Mc/GD58OI8//nja8piYGNq2bcuMGTPo0KFDtudJFR4eTqNGjQCoXbt2htjKlSvH1VdfzejRo+nSpUva8jFjxnDDDTdQtWrVsx5fREQuHCkpsGQJ7NnjkvlgU79+Yz79dFnA1c6nUgu95EhkZGSGZD43Vq5cydatW7nzzjtJTk5Oe5UsWZKrrrqKefPmnfUYZcqUyZBUR0ZGUqtWLTZv3pztPvPnzwege/fuGZZn93ThXM6RGwkJCdStWzctmQcIDQ3NEt/06dOx1tKzZ88M16tly5aEhYXl6HrlxODBg5k2bVra5/vhhx/4/fffGTBgQL4cX0REgkNyskvm9+51o9kEmz17tgNQu/alhIaG+jmac6MW+nwwZ86cbNdFRkaecX358uXPuL569epnXF8QrfMAlSpVOud9d+/eDUDPnj3p2bNnlvX16tU76zHKli2bZVlYWNgZnxKklvJk7ihbsaLvR2nnco7c2LFjh89zZ16Wer2qVavm8zj79u3Ll3h69OjB3/72N8aMGcNLL73EqFGjiImJOecbNxERCT7JybB0qUvmy5f3dzT5b8eOTdxySwMeeugNuncf7O9wzpkSeskR46NQrmjRooSGhnLy5MkMyzMnnKnDSL7xxhu0bNkyy3HCz9P80JUrVwZcgpz6b4Bdu3adl/PlJJ6FCxdmWZ45ntTrNWfOHIoXL55l++h8ah4JDw+nT58+jB07lvvvv58vv/yS5557jpBgGXtMRETyJCUluJN5gJEj/w5AixaBM4mUL0ro5ZyFhoZStWpVVqxYkWF55o6Wl19+OVWqVGHVqlX87W9/K7D4UmvGP//887Qa+tSf/aFZs2Z8+umnLFmyhMaNGwOQkpKSJZ727dtjjGHr1q0+n2jkRurQosePH/e5fuDAgbzxxhvceuuteDwe+vbtm6fziYhIcEjtALtnT3CW2QCsWPEr06Z9Qp8+T1OpUmAP+ayEXvLktttu48033+TVV18lNjaW2bNnZ0lQQ0NDee+99+jevTvHjh3j5ptvply5cuzcuZOffvqJevXqcf/99+d7bA0bNuTmm2/miSeeICkpicaNGzN9+nRmzJgBUOAt0f369eP111/nxhtvZNiwYZQpU4aRI0eSlJSUYbtLLrmEhx56iP79+7NixQpatGhBWFgYmzdvZsaMGTzwwANcc801OTpntWrVKFmyJBMmTKB+/fpERkZSu3ZtypQpA0CtWrWIi4vju+++o3v37tmWI4mIyIXD44EVK2DnzuDsAAtumMq3336EsmUr0KvXE/4OJ8/0bF3yZOjQodx777289dZbdO3alQ0bNjBu3Lgs23Xt2pXZs2ezf/9++vbtS1xcHE8++SR79+7lL3/5y3mL78MPP6Rnz568/PLLdOvWjfXr1/POO+8AUKpUqfN2Xl8iIiL4/vvvueSSS+jfvz99+vTh0ksvzTCSTao333yTd999l++//55bbrmFm266iREjRhAdHU3NmjVzfM6iRYvywQcfsHPnTq6//nquuuoqpk+fnmGb1E656gwrIiKpk0alDk0ZrLZu/ZM//ljGgAEvULx4CX+Hk2fGWuvvGAqN2NhY66vGOdWqVavSJuqRwPXSSy8xdOhQtm/frhZp4Oabb2bFihVpE4Kdb/o7EhEpnKyF33+HTZuCb5x5Xw4c2EOJEmUoUqTgC1b27YO4uPDl1p5omB/HU8mNBLUpU6awbt06GjZ0fy9z587lzTff5K677rqgk3lrLb/88gsLFixgypQpabPsiojIhclaWL36wkjmt23bQJUqMZQpEzydA5TQS1ArUaIEn3/+OS+99BLHjh2jevXqPProozz33HP+Ds2vkpKSaNasGSVKlGDAgAH069fP3yGJiIgfrVsH69dDxYrBncwfP36Ue+5pyvXX38ITT4z0dzj5Rgm9BLW2bdvStm1bf4dR6ISHh6NyOxERAdi4EdauDf6WeYBJk95l//7ddOiQt1HkCht1ihURERG5QG3d6jrBRkdDsE9DcvjwQf7zn9e49tpONGqUs9HiAkWQf3UiIiIi4svOnW6s+fLlITTU39GcfxMmvEli4gEGDXrJ36HkOyX0uaQyBZFzp78fEZHCYd8+WLwYypYFPwzyUuA8Hg9z5nzF9dffQoMGV/g7nHx3AXyF+ado0aIcP36cyMhIf4ciEpCOHz9O0aJF/R2GiMgF7dAhWLAASpWCC+U/ySEhIYwfv5AjRw75O5TzQi30uVChQgW2bdvGsWPH1NIokgvWWo4dO8a2bduoEMwzlYiIFHJHjsCvv0JUFISF+TuagnH48EGSkk5QtGixoBqqMj210OdCyZIlAdi+fTunTp3yczQigaVo0aJUrFgx7e9IREQK1vHjrmW+WDGIiPB3NAVn1Kjn+PHHb/jss98JCwv3dzjnhRL6XC
```

Comment:
A final sentence analyzing the comparison figure would be nice.

Reply:
Added